internal/taskfile/read/taskfile.go

```diff
@@ -44,6 +45,24 @@ func Taskfile(dir string, entrypoint string) (*taskfile.Taskfile, error) {
 		if len(includedTaskfile.Includes) > 0 {
 			return nil, ErrIncludedTaskfilesCantHaveIncludes
 		}
+
+		includedTaskDirectory := filepath.Dir(path)
+		includedTaskfileName := strings.TrimSuffix(filepath.Base(path), filepath.Ext(path))
+		path = filepath.Join(includedTaskDirectory, fmt.Sprintf("%s_%s.yml", includedTaskfileName, runtime.GOOS))
```

Comment:
Hey @evg4b,

Actually, including these files automatically going in the reverse direction I had in mind.

I planned to deprecate the automatic inclusion of Taskfiles by OS in v3, and support templating to something like this would work:

```yaml
includes:
  taskfile: Taskfile_{{OS}}.yml
  namespace: false
```

(The `namespace: false` part needs thought, but we would need a way allow overwriting instead of namespacing).

I propose that you just remove the automatic inclusion for this PR. No need to implement the above now, I can do that in another PR.

What do you think?

Reply:
Sounds good.
Sorry, I did not carefully read the roadmap. I removed this functionality.